modules/ossm-control-plane-deploy.adoc

```diff
@@ -92,9 +89,33 @@ $ oc login https://{HOSTNAME}:6443
 $ oc new-project istio-system
 ----
 
-. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the example found in "Customize the {ProductName} installation". You can customize the values as needed to match your use case.  For production deployments you _must_ change the default Jaeger template.
+. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the following example in "Customize the {ProductName} installation". For production deployments you _must_ change the default Jaeger template.
 
-. Run the following command to deploy the control plane:
+.Example version 2 istio-installation.yaml
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: openshift-operators
+spec:
+  version: v2.0
+  tracing:
+    type: Jaeger
+    sampling: 10000
+  addons:
+    jaeger:
+      install:
+        storage:
+          type: Memory
+    kiali:
+      enabled: true
+    grafana:
+      enabled: true
+----
+
+. Run the following command to deploy the control plane, where istio-installation.yaml includes the full path to your file.
```

Comment:
Should istio-installation.yaml be a <variable> since customers are supposed to provide the full path?

Reply:
Also, in light of the upgrade needing a new namespace, should we make istio-system a <variable> too?

Reply:
I agree that the istio installation.yaml should be a variable. I made that change. 

I think for this part of the installation, istio-system shouldn't be a variable. I think the upgrade is a special case and doesn't have to affect this install path.